tests/foreman/ui/test_docker.py

```diff
@@ -1338,19 +1379,39 @@ def test_update_registry_username(self):
 
         @Assert: the external registry is updated with the new username
 
-        @Status: Manual
-
         """
+        with Session(self.browser) as session:
+            name = gen_string('utf8')
+            make_registry(
+                session,
+                name=name,
+                url=gen_url(subdomain=gen_string('alpha')),
+                username=gen_string('alphanumeric'),
+            )
+            self.assertIsNotNone(self.registry.search(name))
+
+            new_username = gen_string('utf8')
+            self.registry.update(name, new_username=new_username)
+            self.registry.search(name).click()
+            self.assertIsNotNone(self.registry.wait_until_element(
+                locators['registry.username']).text, new_username)
 
-    @stubbed()
     @run_only_on('sat')
     def test_delete_registry(self):
-        """@Test: Create an external docker registry
+        """@Test: Create an external docker registry and then delete it
 
         @Feature: Docker
 
-        @Assert: the external registry is created
-
-        @Status: Manual
+        @Assert: The external registry is deleted successfully
 
         """
+        with Session(self.browser) as session:
+            for name in valid_data_list():
+                with self.subTest(name):
+                    make_registry(
+                        session,
+                        name=name,
+                        url=gen_url(subdomain=gen_string('alpha')),
+                        description=gen_string('utf8'),
+                    )
+                    self.registry.delete(name)
```

Comment:
That is actually result of my delete functionality re-factor procedure. @sghai mentioned that assertion should be always performed on test level, but it is pretty hard to implement that approach without huge amount of redundant code
You can refer to https://github.com/SatelliteQE/robottelo/pull/2809 for more details about improvement